swupd/packs.go

```diff
@@ -295,7 +295,8 @@ func WritePack(w io.Writer, fromManifest, toManifest *Manifest, outputDir, chroo
 func copyFromDelta(tw *tar.Writer, delta *Delta) (fallback bool, err error) {
 	f, err := os.Open(delta.Path)
 	if err != nil {
-		return true, err
+		log.Debug(log.Mixer, err.Error())
+		return true, nil
```

Comment:
Maybe we should return error as `nil` only if the err is `os.IsNotExist(err)`. For any other error condition it should return the valid error.